src/router/routeActionCreators.js

```diff
@@ -0,0 +1,9 @@
+export const LOCATION_CHANGE = '@@router/LOCATION_CHANGE';
+export const SERVER_RENDER = '@@router/INITIAL_RENDER';
```

Comment:
Both [Redux](https://github.com/reactjs/redux/blob/c9ad6a40fe501919d359b910ea7c7f34532e11d9/examples/todos-flow/src/types/index.js#L11) and [react-router-redux](https://github.com/reactjs/react-router-redux/blob/a304bda820ba57f68961035eff1950e618047b81/src/reducer.js#L5) use this convention for actions that are triggered by middleware rather than the UI, so I've mirrored it here - specifically `@@router/LOCATION_CHANGE` is the same as the action from react-router-redux, and the purpose of it is identical here.